controllers/cluster_controller.go

```diff
@@ -166,10 +167,18 @@ func (r *FoundationDBClusterReconciler) Reconcile(ctx context.Context, request c
 			continue
 		}
 
+		if requeue.DelayedRequeue {
```

Comment:
I don't make use of that yet, but the plan is to use this for delayed requeue e.g. when we want to iterate further but in the end queue the cluster again.